### DISCUSSION TITLE
How to set "remember me"

### DISCUSSION THREAD

**Discussion by adamzerner** [supabase/supabase#11695] [Questions]
I am using [`signInWithPassword`](https://supabase.com/docs/reference/javascript/auth-signinwithpassword) in a Next.js app following that mostly follows the [guide](https://supabase.com/docs/guides/getting-started/tutorials/with-nextjs). This has "remember me" functionality -- ie. `useSession` gives me the users session even after closing the browser and re-opening it -- but I'd like to present my users with the ability to not have this "remember me" functionality. Is there a way to do that?

**Comment by rudolphmax** [comment]
Hi Adam!
Have you found a solution to this? I'd be really interested as well.

**Reply by nicitaacom** [reply]
I also wondered - I got answer from supabase 

![image](https://github.com/supabase/supabase/assets/39565703/d2c52241-fe88-4a20-ab40-8513c0508f93)

![image](https://github.com/supabase/supabase/assets/39565703/a3d7da59-49c6-4cfe-bce1-0fd827d00840)

**Comment by nicitaacom** [comment]
So waiting until supabase release this feature

**Reply by rudolphmax** [reply]
Okay. How disappointing. 
Thanks for the answer!
